pebbles/views/blueprints.py

```diff
@@ -56,7 +56,7 @@ def get(self):
         # sort the results based on the group name first and then by blueprint name
         query = query.join(Group, Blueprint.group).order_by(Group.name).order_by(Blueprint.name)
         all_blueprints = []
-        if args.get('show_all'):
+        if args.get('show_all') and user.is_admin:
```

Comment:
We could modify apply_rules_blueprints(user) instead, so that the access policy is global.

Reply:
And have a unit test that checks this behaviour, too

Reply:
The code has been moved to rules.py, and the unit tests actually cover that code fully already.